app/src/main/java/com/termux/app/ExtraKeysView.java

```diff
@@ -71,9 +75,15 @@ static void sendKey(View view, String keyName) {
             case "↓":
                 keyCode = KeyEvent.KEYCODE_DPAD_DOWN;
                 break;
+            case "↲":
```

Comment:
Add case ENTER too, of course the displayed chars must be "↲" but if people write ENTER in termux.properties it should work as expected.